### DISCUSSION TITLE
It is possible config previous/next section/page for current dicrectory

### DISCUSSION THREAD

**Discussion by tatdatpham** [squidfunk/mkdocs-material#4433] [Q&A] [answered]
Hi everyone,

I want to customize previous/next button in footer. Currently, the function `page.previous_page` & `page.next_page` are setting for all page in the site.I want to set only for the current category (or directory) that the current page belongs.
For example, my sitemap is
```
https://site.com/directory-one/page-01
https://site.com/directory-one/page-02
https://site.com/directory-one/page-03
https://site.com/cheatsheets/git
https://site.com/cheatsheets/bash
https://site.com/cheatsheets/python
```
**Scenario 01**:
My current page is
```
https://site.com/cheatsheets/git
```
My expectation :
 - `page.previous_page` -> null (do not show anything because current page is first page in this category)
 - `page.next_page` -> `https://site.com/cheatsheets/bash`
 
**Scenario 02**:
My current page is
```
https://site.com/directory-one/page-03
```
My expectation :
 - `page.previous_page` -> `https://site.com/directory-one/page-02`
 - `page.next_page` -> null (do not show anything because current page is last page in this category)
 
Thank you for help!

**Comment by squidfunk** [comment, marked as answer]
Thanks for asking. It's currently not possible, but it might be a great idea for a plugin!

**Reply by tatdatpham** [reply]
Thank you for your suggestion. I just heard about `mkdocs` a few days ago.  The truth is I knew `mkdocs-material` before` mkdocs` 😄 . I will try to start will the plugin.

**Comment by rsbillington** [comment]
Hi there,

New to MkDocs and Material and was thinking along similar lines for my new layout/navigation, the top-level sections are not related, and I wanted to just drop the Next/Previous link on the first and last page in each top-level section. It is possible, although manually on the first and last pages of each section, (although others may know a better way):

Add the `attr_list` extension in the `config.yml`:
```yaml
markdown_extensions:
  - attr_list
```
On the first page on a main section hide the Previous Link, I just put the CSS inline in the main body of the page markdown:

```markdown
---
title: First Page in Section
----
<style>
  .md-footer__link--prev:not([hidden]) { display: none }
</style>
```
And then on the last page of the section:
```markdown
---
title: Last Page in Section
----
<style>
  .md-footer__link--next:not([hidden]) { display: none }
</style>
```

I am not sure if this will "break" anything in the future, but it seems to work in my setup! I just need to remember when I add a new page that then becomes the new last page to update the previous page, no big deal unless the site is massive, I guess.
